docs/AddingAndAlteringTemplates.md

```diff
@@ -26,6 +26,9 @@ object MyLovelyHorseEdition extends EditionDefinitionWithTemplate {
   ...
 ```
 
+NOTE: the `edition` property of your object must be a hypen case version of your object name otherwise the routing won't
```

Comment:
```suggestion
NOTE: the `edition` property of your object must be a hyphen case version of your object name otherwise the routing won't
```

Reply:
And also... what?  Why must it match?

Reply:
It's your code! This caught me out when I initially tried to test this - I guess the dynamic routing thing defaults a route for ClassName to class-name